src/DfuError.js

```diff
@@ -163,34 +204,33 @@ export class DfuError extends Error {
     }
 
     getErrorMessage(code) {
-        let errorMsg;
         const errorType = code >> 8;
 
         debug(`Error type is ${errorType}.`);
 
-        errorMsg = ErrorTypes[errorType];
-        if (!errorMsg) {
+        this.errorMsg = ErrorTypes[errorType];
```

Comment:
Is `.errorMsg` accessed from anywhere outside this method? After a quick search of the code in this PR I don't see the reason for this change.

Reply:
I gather the reason is that `this` should be used or the member should be declared static, either way is correct.

Reply:
This is used here due to fixing eslint warning. I can update it if we have better solution or disable eslint.

Reply:
I suggest to revert this change and use a static method instead if we don't plan to use `this.errorMsg`, it would be a cleaner solution.